src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
 }
+
+.header {
+  display: flex;
+  width: auto;
+  height: 60px;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.logo {
+  position: absolute;
+}
+
+.links {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+  display: block;
+}
+
+.links:hover,
+.is-active {
+  color: #00ACDC;
```

Comment:
Create a variable for repeated color and use it everywhere

Reply:
Not fixed